src/components/Divider/Divider.stories.tsx

```diff
@@ -35,7 +38,30 @@ export default {
                 ),
             },
         },
+        vertical: {
+            control: {
+                type: 'boolean',
+            },
+        },
     },
 } as Meta;
 
-export const Divider: StoryFn<DividerProps> = (args: DividerProps) => <DividerComponent {...args} />;
+export const Divider: StoryFn<DividerProps> = (args: DividerProps) => {
```

Comment:
It would be nice to have a story with the vertical divider between elements with different amount of content to see its behaviour on this type of use case. WDYT?